OTRExporter/OTRExporter/Makefile

```diff
@@ -39,13 +39,13 @@ D_FILES   := $(O_FILES:%.o=%.d)
 LIB       := OTRExporter.a
 
 INC_DIRS := $(addprefix -I, \
-	../../ZAPD/ZAPD \
-	../../ZAPD/lib/tinyxml2 \
-	../../ZAPD/lib/libgfxd \
-	../../ZAPD/ZAPDUtils \
-	../../OtrLib/otrlib \
-	../../OtrLib/otrlib/Lib/spdlog/include \
-	../../OtrLib/otrlib/Lib/Fast3D/U64 \
+	../../ZAPDTR/ZAPD \
```

Comment:
Is this  just changing paths for the sake of changing paths? Is there anything necessary about this refactor to getting SOH on Linux? If not it might be better for it to live in a separate PR.

Reply:
This is because I wrote the old Makefile a while ago when libultraship and ZAPDTR were still named respectively "OtrLib" and "ZAPD".